diff -ru tmp/cisco-mibs/v2/CISCO-ENTITY-VENDORTYPE-OID-MIB.my new/cisco/CISCO-ENTITY-VENDORTYPE-OID-MIB.my
--- tmp/cisco-mibs/v2/CISCO-ENTITY-VENDORTYPE-OID-MIB.my	2026-04-19 17:36:13.407318480 +0200
+++ new/cisco/CISCO-ENTITY-VENDORTYPE-OID-MIB.my	2026-04-19 20:30:10.449570398 +0200
@@ -5485,7 +5485,7 @@
 cevMAs5800Dspm6c549	OBJECT IDENTIFIER ::= { cevModuleAS5800Type 9 }		-- Digital Signal Processor Module with 6 C549's Voice Feature Card
 cevMAs5800Dsp		OBJECT IDENTIFIER ::= { cevModuleAS5800Type 10 }	-- Digital Signal Processor Module Voice Feature Card
 
-cevModuleC9KVA100L8P        OBJECT IDENTIFIER ::= { cevModuleCat9000Type 6 }	-- Catalyst 9300 Argon 8 Port switch
+-- cevModuleC9KVA100L8P        OBJECT IDENTIFIER ::= { cevModuleCat9000Type 6 } --	-- Catalyst 9300 Argon 8 Port switch
 
 cevModuleFabricType		OBJECT IDENTIFIER ::= { cevModule 14 }
 cev12012Sfc	OBJECT IDENTIFIER ::= { cevModuleFabricType 3 }  -- 12012 Switch Fabric Card
diff -ru tmp/cisco-mibs/v2/CISCO-FC-FE-MIB.my new/cisco/CISCO-FC-FE-MIB.my
--- tmp/cisco-mibs/v2/CISCO-FC-FE-MIB.my	2026-04-19 17:36:13.419318350 +0200
+++ new/cisco/CISCO-FC-FE-MIB.my	2026-04-19 20:29:23.868076042 +0200
@@ -7334,7 +7334,7 @@
     DESCRIPTION
         "Write access is not required."
 
-    OBJECT          TransceiverPowerControl
+    OBJECT          fcIfTransceiverPowerControl
     MIN-ACCESS      read-only
     DESCRIPTION
         "Write access is not required."
@@ -7936,7 +7936,7 @@
                         fcIfTransceiverPower,
                         fcIfTransceiverPowerControl,
                         fcIfSysTransceiverPowerControlCapability,
-                        fcIfSysTransceiverPowerControl,
+                        fcIfSysTransceiverPowerControl
                     }
     STATUS          current
     DESCRIPTION
diff -ru tmp/cisco-mibs/v2/CISCO-IDSL-LINE-MIB.my new/cisco/CISCO-IDSL-LINE-MIB.my
--- tmp/cisco-mibs/v2/CISCO-IDSL-LINE-MIB.my	2023-03-27 16:00:52.083399258 +0200
+++ new/cisco/CISCO-IDSL-LINE-MIB.my	2026-04-19 17:36:18.600262345 +0200
@@ -26,7 +26,6 @@
     PerfCurrentCount,
     PerfIntervalCount                   FROM PerfHist-TC-MIB
     SnmpAdminString                     FROM SNMP-FRAMEWORK-MIB
-    AdslLineProfileName,
     adslLineConfProfileName             FROM ADSL-LINE-MIB
     ;
 
@@ -194,8 +193,8 @@
 
 CIdslLineEntry ::= SEQUENCE {
     cIdslLineSpecific         VariablePointer,
-    cIdslLineConfProfile      AdslLineProfileName,
-    cIdslLineAlarmConfProfile AdslLineProfileName
+    cIdslLineConfProfile      SnmpAdminString,
+    cIdslLineAlarmConfProfile SnmpAdminString
 }
 
 cIdslLineSpecific OBJECT-TYPE
@@ -212,7 +211,7 @@
     ::= { cIdslLineEntry 1 }
 
 cIdslLineConfProfile OBJECT-TYPE
-    SYNTAX      AdslLineProfileName
+    SYNTAX      SnmpAdminString
     MAX-ACCESS  read-write
     STATUS      current
     DESCRIPTION
@@ -231,7 +230,7 @@
     ::= { cIdslLineEntry 2 }
 
 cIdslLineAlarmConfProfile OBJECT-TYPE
-    SYNTAX      AdslLineProfileName
+    SYNTAX      SnmpAdminString
     MAX-ACCESS  read-write
     STATUS      current
     DESCRIPTION
diff -ru tmp/cisco-mibs/v2/CISCO-IEEE-CFM-MIB.my new/cisco/CISCO-IEEE-CFM-MIB.my
--- tmp/cisco-mibs/v2/CISCO-IEEE-CFM-MIB.my	2023-03-27 16:00:52.084399246 +0200
+++ new/cisco/CISCO-IEEE-CFM-MIB.my	2026-04-19 17:36:18.603262313 +0200
@@ -19,7 +19,7 @@
 -- this MIB.
 
 
-IEEE8021-CFM-MIB DEFINITIONS ::= BEGIN
+CISCO-IEEE8021-CFM-MIB DEFINITIONS ::= BEGIN
 
 -- ******************************************************************
 -- IEEE P802.1ag(TM) CFM MIB
diff -ru tmp/cisco-mibs/v2/CISCO-NAT-CGN-EXT-MIB.my new/cisco/CISCO-NAT-CGN-EXT-MIB.my
--- tmp/cisco-mibs/v2/CISCO-NAT-CGN-EXT-MIB.my	2023-03-27 16:00:52.215397755 +0200
+++ new/cisco/CISCO-NAT-CGN-EXT-MIB.my	2026-04-19 17:36:18.601262334 +0200
@@ -32,7 +32,7 @@
     natInstanceIndex,
     natInstanceEntry,
     natCountersEntry
-        FROM NAT-MIB
+        FROM XXX-NAT-MIB
     ciscoMgmt
         FROM CISCO-SMI;
 
diff -ru tmp/cisco-mibs/v2/CISCO-RTTMON-TC-MIB.my new/cisco/CISCO-RTTMON-TC-MIB.my
--- tmp/cisco-mibs/v2/CISCO-RTTMON-TC-MIB.my	2023-03-27 16:00:52.296396834 +0200
+++ new/cisco/CISCO-RTTMON-TC-MIB.my	2026-04-19 17:36:18.601262334 +0200
@@ -428,7 +428,7 @@
 
         The value 'fabricPathEcho' will cause the RTT application to
         perform delay performance measurment and verify connectivity in
-        a Fabric Path Network."
+        a Fabric Path Network.
 
         NOTE:  The 'pathJitter' time delay operation is a heuristic
                measurement because an intermediate hop may forward
diff -ru tmp/cisco-mibs/v2/CISCO-SDSL-LINE-MIB.my new/cisco/CISCO-SDSL-LINE-MIB.my
--- tmp/cisco-mibs/v2/CISCO-SDSL-LINE-MIB.my	2023-03-27 16:00:52.298396811 +0200
+++ new/cisco/CISCO-SDSL-LINE-MIB.my	2026-04-19 17:36:18.602262323 +0200
@@ -22,7 +22,6 @@
     MODULE-COMPLIANCE,
     OBJECT-GROUP                        FROM SNMPv2-CONF
     ciscoMgmt                           FROM CISCO-SMI
-    AdslLineProfileName,
     adslLineConfProfileName,
     adslLineAlarmConfProfileName        FROM ADSL-LINE-MIB
     ifIndex                             FROM IF-MIB
@@ -267,8 +266,8 @@
     cSdslLineCoding             INTEGER,
     cSdslLineFraming            INTEGER,
     cSdslLineSpecific           VariablePointer,
-    cSdslLineConfProfile        AdslLineProfileName,
-    cSdslLineAlarmConfProfile   AdslLineProfileName
+    cSdslLineConfProfile        SnmpAdminString,
+    cSdslLineAlarmConfProfile   SnmpAdminString
 }
 
 cSdslLineCoding OBJECT-TYPE
@@ -338,7 +337,7 @@
 
 
 cSdslLineConfProfile OBJECT-TYPE
-    SYNTAX      AdslLineProfileName
+    SYNTAX      SnmpAdminString
     MAX-ACCESS  read-write
     STATUS      current
     DESCRIPTION
@@ -357,7 +356,7 @@
     ::= { cSdslLineEntry 4 }
 
 cSdslLineAlarmConfProfile OBJECT-TYPE
-    SYNTAX      AdslLineProfileName
+    SYNTAX      SnmpAdminString
     MAX-ACCESS  read-write
     STATUS      current
     DESCRIPTION
diff -ru tmp/cisco-mibs/v2/CISCO-SIP-UA-MIB.my new/cisco/CISCO-SIP-UA-MIB.my
--- tmp/cisco-mibs/v2/CISCO-SIP-UA-MIB.my	2023-10-27 09:23:45.433984413 +0200
+++ new/cisco/CISCO-SIP-UA-MIB.my	2026-04-19 17:36:18.607262269 +0200
@@ -4591,12 +4591,6 @@
                not provide any timer configuration and simply adhere to
                what is prescribed in the standard."
 
-         GROUP ciscoSipRetryConfigGroupRev5
-         DESCRIPTION
-              "This group is optional. A SIP user agent can elect to
-               not provide any timer configuration and simply adhere to
-               what is prescribed in the standard."
-
          GROUP ciscoSipRetryStatsGroupRev4
          DESCRIPTION
               "This group is optional.  If implemented, the
diff -ru tmp/cisco-mibs/v2/CISCO-ST-TC.my new/cisco/CISCO-ST-TC.my
--- tmp/cisco-mibs/v2/CISCO-ST-TC.my	2024-09-17 20:14:11.515621698 +0200
+++ new/cisco/CISCO-ST-TC.my	2026-04-19 17:36:18.610262237 +0200
@@ -362,7 +362,7 @@
           thirtyTwoG  (12) - 32GBit.
           autoMaxThirtyTwoG (13) - Negotiate to determine the
                              speed automatically upto a
-                             maximum of 32Gbit."
+                             maximum of 32Gbit.
           fiftyG      (14) - 50GBit.
           sixtyFourG  (15) - 64GBit.
           autoMaxSixtyFourG (16) - Negotiate to determine the
diff -ru tmp/cisco-mibs/v2/CISCO-SWITCH-QOS-MIB.my new/cisco/CISCO-SWITCH-QOS-MIB.my
--- tmp/cisco-mibs/v2/CISCO-SWITCH-QOS-MIB.my	2023-03-27 16:00:52.380395877 +0200
+++ new/cisco/CISCO-SWITCH-QOS-MIB.my	2026-04-19 17:36:18.604262302 +0200
@@ -3032,12 +3032,12 @@
 
 csqHwSharedPoolStatsDirection OBJECT-TYPE
     SYNTAX          INTEGER  {
-                        inputStats_ingressStraight(1),
-                        inputStats_ingressHairpin(2),
-                        inputStats_egress(3),
-                        outputStats_ingressStraight(4),
-                        outputStats_ingressHairpin(5),
-                        outputStats_egress(6)
+                        inputStats-ingressStraight(1),
+                        inputStats-ingressHairpin(2),
+                        inputStats-egress(3),
+                        outputStats-ingressStraight(4),
+                        outputStats-ingressHairpin(5),
+                        outputStats-egress(6)
                     }
     MAX-ACCESS      not-accessible
     STATUS          current
