diff -ru auto/mibs/v2/CISCO-FEATURE-CONTROL-MIB.my cisco/CISCO-FEATURE-CONTROL-MIB.my
--- auto/mibs/v2/CISCO-FEATURE-CONTROL-MIB.my	2015-05-27 21:03:17.000000000 +0200
+++ cisco/CISCO-FEATURE-CONTROL-MIB.my	2016-06-07 13:14:05.000000000 +0200
@@ -32,7 +32,7 @@
 
 
 ciscoFeatureCtrlMIB MODULE-IDENTITY
-    LAST-UPDATED    "201412240000"
+    LAST-UPDATED    "201412240000Z"
     ORGANIZATION    "Cisco Systems, Inc."
     CONTACT-INFO
             "Cisco Systems
diff -ru auto/mibs/v2/CISCO-IDSL-LINE-MIB.my cisco/CISCO-IDSL-LINE-MIB.my
--- auto/mibs/v2/CISCO-IDSL-LINE-MIB.my	2000-07-01 09:00:00.000000000 +0200
+++ cisco/CISCO-IDSL-LINE-MIB.my	2016-06-07 13:15:04.000000000 +0200
@@ -26,7 +26,6 @@
     PerfCurrentCount,
     PerfIntervalCount                   FROM PerfHist-TC-MIB
     SnmpAdminString                     FROM SNMP-FRAMEWORK-MIB
-    AdslLineProfileName,
     adslLineConfProfileName             FROM ADSL-LINE-MIB
     ;
 
@@ -194,8 +193,8 @@
 
 CIdslLineEntry ::= SEQUENCE {
     cIdslLineSpecific         VariablePointer,
-    cIdslLineConfProfile      AdslLineProfileName,
-    cIdslLineAlarmConfProfile AdslLineProfileName
+    cIdslLineConfProfile      SnmpAdminString,
+    cIdslLineAlarmConfProfile SnmpAdminString
 }
 
 cIdslLineSpecific OBJECT-TYPE
@@ -212,7 +211,7 @@
     ::= { cIdslLineEntry 1 }
 
 cIdslLineConfProfile OBJECT-TYPE
-    SYNTAX      AdslLineProfileName
+    SYNTAX      SnmpAdminString
     MAX-ACCESS  read-write
     STATUS      current
     DESCRIPTION
@@ -231,7 +230,7 @@
     ::= { cIdslLineEntry 2 }
 
 cIdslLineAlarmConfProfile OBJECT-TYPE
-    SYNTAX      AdslLineProfileName
+    SYNTAX      SnmpAdminString
     MAX-ACCESS  read-write
     STATUS      current
     DESCRIPTION
diff -ru auto/mibs/v2/CISCO-NAT-CGN-EXT-MIB.my cisco/CISCO-NAT-CGN-EXT-MIB.my
--- auto/mibs/v2/CISCO-NAT-CGN-EXT-MIB.my	2016-02-17 10:23:13.000000000 +0100
+++ cisco/CISCO-NAT-CGN-EXT-MIB.my	2016-06-07 13:42:45.000000000 +0200
@@ -32,7 +32,7 @@
     natInstanceIndex,
     natInstanceEntry,
     natCountersEntry
-        FROM NAT-MIB
+        FROM XXX-NAT-MIB
     ciscoMgmt
         FROM CISCO-SMI;
 
diff -ru auto/mibs/v2/CISCO-RTTMON-TC-MIB.my cisco/CISCO-RTTMON-TC-MIB.my
--- auto/mibs/v2/CISCO-RTTMON-TC-MIB.my	2012-06-01 23:06:31.000000000 +0200
+++ cisco/CISCO-RTTMON-TC-MIB.my	2016-06-07 13:35:05.000000000 +0200
@@ -90,6 +90,26 @@
 
 -- Textual Conventions
 
+RttMonScheduleStartType ::= TEXTUAL-CONVENTION
+    STATUS          current
+    DESCRIPTION
+	"This object specifies the start-time option to be used for
+	scheduling the start time of RTT operation.
+	The following list defines the start-time options
+	that can be configured for the probe:
+	    pending(1)      pending state.
+	    now(2)          schedule to immediately start the probe.
+	    random(3)       schedule start time of probe at random time over a range.
+	    after(4)        schedule start time of probe after a certain amount of time from now.
+	    specific(5)     schedule start time of probe at a given specific time interval"
+    SYNTAX          INTEGER  {
+			pending(1),
+			now(2),
+			random(3),
+			after(4),
+			specific(5)
+		    }
+
 RttReset ::= TEXTUAL-CONVENTION
     STATUS          current
     DESCRIPTION
diff -ru auto/mibs/v2/CISCO-SDSL-LINE-MIB.my cisco/CISCO-SDSL-LINE-MIB.my
--- auto/mibs/v2/CISCO-SDSL-LINE-MIB.my	2001-12-04 02:11:06.000000000 +0100
+++ cisco/CISCO-SDSL-LINE-MIB.my	2016-06-07 13:15:27.000000000 +0200
@@ -22,7 +22,6 @@
     MODULE-COMPLIANCE,
     OBJECT-GROUP                        FROM SNMPv2-CONF
     ciscoMgmt                           FROM CISCO-SMI
-    AdslLineProfileName,
     adslLineConfProfileName,
     adslLineAlarmConfProfileName        FROM ADSL-LINE-MIB
     ifIndex                             FROM IF-MIB
@@ -267,8 +266,8 @@
     cSdslLineCoding             INTEGER,
     cSdslLineFraming            INTEGER,
     cSdslLineSpecific           VariablePointer,
-    cSdslLineConfProfile        AdslLineProfileName,
-    cSdslLineAlarmConfProfile   AdslLineProfileName
+    cSdslLineConfProfile        SnmpAdminString,
+    cSdslLineAlarmConfProfile   SnmpAdminString
 }
 
 cSdslLineCoding OBJECT-TYPE
@@ -338,7 +337,7 @@
 
 
 cSdslLineConfProfile OBJECT-TYPE
-    SYNTAX      AdslLineProfileName
+    SYNTAX      SnmpAdminString
     MAX-ACCESS  read-write
     STATUS      current
     DESCRIPTION
@@ -357,7 +356,7 @@
     ::= { cSdslLineEntry 4 }
 
 cSdslLineAlarmConfProfile OBJECT-TYPE
-    SYNTAX      AdslLineProfileName
+    SYNTAX      SnmpAdminString
     MAX-ACCESS  read-write
     STATUS      current
     DESCRIPTION
diff --git a/cisco/CISCO-IEEE-CFM-MIB.my b/cisco/CISCO-IEEE-CFM-MIB.my
index 698f7b9..188ea68 100644
--- a/cisco/CISCO-IEEE-CFM-MIB.my
+++ b/cisco/CISCO-IEEE-CFM-MIB.my
@@ -19,7 +19,7 @@
 -- this MIB.
 
 
-IEEE8021-CFM-MIB DEFINITIONS ::= BEGIN
+CISCO-IEEE8021-CFM-MIB DEFINITIONS ::= BEGIN
 
 -- ******************************************************************
 -- IEEE P802.1ag(TM) CFM MIB
diff --git a/cisco/CISCO-SWITCH-QOS-MIB.my b/cisco/CISCO-SWITCH-QOS-MIB.my
index 28b1b63..ddc51f1 100644
--- a/cisco/CISCO-SWITCH-QOS-MIB.my
+++ b/cisco/CISCO-SWITCH-QOS-MIB.my
@@ -3032,12 +3032,12 @@ csqHwSharedPoolUsageInstNo OBJECT-TYPE
 
 csqHwSharedPoolStatsDirection OBJECT-TYPE
     SYNTAX          INTEGER  {
-                        inputStats_ingressStraight(1),
-                        inputStats_ingressHairpin(2),
-                        inputStats_egress(3),
-                        outputStats_ingressStraight(4),
-                        outputStats_ingressHairpin(5),
-                        outputStats_egress(6)
+                        inputStats-ingressStraight(1),
+                        inputStats-ingressHairpin(2),
+                        inputStats-egress(3),
+                        outputStats-ingressStraight(4),
+                        outputStats-ingressHairpin(5),
+                        outputStats-egress(6)
                     }
     MAX-ACCESS      not-accessible
     STATUS          current
