diff --git a/cisco-lwapp/AIRESPACE-WIRELESS-MIB.my b/cisco-lwapp/AIRESPACE-WIRELESS-MIB.my
index dd84157..11125bc 100644
--- a/cisco-lwapp/AIRESPACE-WIRELESS-MIB.my
+++ b/cisco-lwapp/AIRESPACE-WIRELESS-MIB.my
@@ -2437,7 +2437,7 @@ bsnMobileStationPolicyType OBJECT-TYPE
                         wpa2vff(3),
                         notavailable(4),
                         unknown(5),
-                        wpa2_3(6),
+                        wpa2-3(6),
                         wpaOsen(7),
                         wpaOsenWithinRsn(8)
                     }
@@ -4696,8 +4696,8 @@ bsnAPIfType OBJECT-TYPE
                         dot11b(1),
                         dot11a(2),
                         uwb(4),
-                        dot11_6ghz(6),
-                        dot11_xor_5_6ghz(7)
+                        dot11-6ghz(6),
+                        dot11-xor-5-6ghz(7)
                     }
     MAX-ACCESS      read-only
     STATUS          current
diff -ruw ../cisco-lwapp/CISCO-LWAPP-DOT11-CLIENT-MIB.my new/cisco-lwapp/CISCO-LWAPP-DOT11-CLIENT-MIB.my
--- ../cisco-lwapp/CISCO-LWAPP-DOT11-CLIENT-MIB.my	2023-07-26 12:23:38.133339987 +0200
+++ new/cisco-lwapp/CISCO-LWAPP-DOT11-CLIENT-MIB.my	2023-03-27 16:00:52.159398393 +0200
@@ -1189,7 +1189,7 @@
                          wpa2(3),
                          wpa2vff(4),
                          notavailable(5),
-                         wpa2_3(6),
+                         wpa2-3(6),
                          wpaOsen(7),
                          wpaOsenWithinRsn(8)
                      }
diff --git a/cisco-lwapp/CISCO-LWAPP-AP-MIB.my b/cisco-lwapp/CISCO-LWAPP-AP-MIB.my
index 9f51e4a..b955226 100644
--- a/cisco-lwapp/CISCO-LWAPP-AP-MIB.my
+++ b/cisco-lwapp/CISCO-LWAPP-AP-MIB.my
@@ -4244,7 +4244,7 @@ cLApWlanStatsOnlineUserNum OBJECT-TYPE
 -- *  AP slot WLAN Stats.
 -- ********************************************************************
 
-cLApWlanSlotStatsTable OBJECT-TYPE
+cLApSlotWlanStatsTable OBJECT-TYPE
     SYNTAX          SEQUENCE OF CLApSlotWlanStatsEntry
     MAX-ACCESS      not-accessible
     STATUS          current
diff --git a/cisco-lwapp/CISCO-LWAPP-REAP-MIB.my b/cisco-lwapp/CISCO-LWAPP-REAP-MIB.my
index 976e031..6cedef4 100644
--- a/cisco-lwapp/CISCO-LWAPP-REAP-MIB.my
+++ b/cisco-lwapp/CISCO-LWAPP-REAP-MIB.my
@@ -2029,7 +2029,7 @@ cLReapGroupPMKAPPropagation OBJECT-TYPE
          controller to only few APs and from those AP to other APs in
          site tag.
          A value of 'pmkDistDCDS' specifies PMK distribution between 
-         APs in site tag.
+         APs in site tag."
     
     DEFVAL          { pmkDistCntrToAp } 
     ::= { cLReapGroupConfigEntry 57 }
diff --git a/cisco-fp/CISCO-FIREPOWER-AP-AAA-MIB.my b/cisco-fp/CISCO-FIREPOWER-AP-AAA-MIB.my
index ffeed30..85460f2 100644
--- a/cisco-fp/CISCO-FIREPOWER-AP-AAA-MIB.my
+++ b/cisco-fp/CISCO-FIREPOWER-AP-AAA-MIB.my
@@ -65,7 +65,6 @@ IMPORTS
     CfprApAaaLdapEpFsmStageName,
     CfprApAaaLdapGroupRuleAuthorization,
     CfprApAaaLdapGroupRuleTraversal,
-    CfprApAaaLdapSsl,
     CfprApAaaLdapVendor,
     CfprApAaaNoRolePolicy,
     CfprApAaaPwdPolicy,
diff --git a/cisco-fp/CISCO-FIREPOWER-AP-COMM-MIB.my b/cisco-fp/CISCO-FIREPOWER-AP-COMM-MIB.my
index 8e6df2f..cbb515e 100644
--- a/cisco-fp/CISCO-FIREPOWER-AP-COMM-MIB.my
+++ b/cisco-fp/CISCO-FIREPOWER-AP-COMM-MIB.my
@@ -55,7 +55,6 @@ IMPORTS
     CfprApCommNtpProviderAdminState,
     CfprApCommProtocol,
     CfprApCommShellProto,
-    CfprApCommSnmpAdminState,
     CfprApCommSnmpAuth,
     CfprApCommSnmpConfigState,
     CfprApCommSnmpNotificationType,
diff --git a/cisco-fp/CISCO-FIREPOWER-AP-EQUIPMENT-MIB.my b/cisco-fp/CISCO-FIREPOWER-AP-EQUIPMENT-MIB.my
index 3582c06..5eb2ddb 100644
--- a/cisco-fp/CISCO-FIREPOWER-AP-EQUIPMENT-MIB.my
+++ b/cisco-fp/CISCO-FIREPOWER-AP-EQUIPMENT-MIB.my
@@ -135,15 +135,12 @@ IMPORTS
     CfprApEquipmentPsuInputSrc,
     CfprApEquipmentPsuInputStatsHistThresholded,
     CfprApEquipmentPsuInputStatsThresholded,
-    CfprApEquipmentPsuOperability,
     CfprApEquipmentPsuOutputStatsHistThresholded,
     CfprApEquipmentPsuOutputStatsThresholded,
-    CfprApEquipmentPsuPower,
     CfprApEquipmentPsuStateQualifier,
     CfprApEquipmentPsuStatsHistThresholded,
     CfprApEquipmentPsuStatsThresholded,
     CfprApEquipmentPsuType,
-    CfprApEquipmentPsuVoltage,
     CfprApEquipmentPsuVoltageType,
     CfprApEquipmentRackUnitFanStatsHistThresholded,
     CfprApEquipmentRackUnitFanStatsThresholded,
diff --git a/cisco-fp/CISCO-FIREPOWER-AP-FABRIC-MIB.my b/cisco-fp/CISCO-FIREPOWER-AP-FABRIC-MIB.my
index feaa6ad..bf593a4 100644
--- a/cisco-fp/CISCO-FIREPOWER-AP-FABRIC-MIB.my
+++ b/cisco-fp/CISCO-FIREPOWER-AP-FABRIC-MIB.my
@@ -111,12 +111,10 @@ IMPORTS
     CfprApFabricEthEstcPortMode,
     CfprApFabricEthEstcTransport,
     CfprApFabricEthEstcType,
-    CfprApFabricEthLanEpAdminSpeed,
     CfprApFabricEthLanEpAdminState,
     CfprApFabricEthLanEpSpeedCap,
     CfprApFabricEthLanEpVlanStatus,
     CfprApFabricEthLanPcAdminState,
-    CfprApFabricEthLanPcEpAdminState,
     CfprApFabricEthLanPcLacpDetach,
     CfprApFabricEthLanPcSpeedCap,
     CfprApFabricEthLanPcTransport,
diff --git a/cisco-fp/CISCO-FIREPOWER-AP-PROCESSOR-MIB.my b/cisco-fp/CISCO-FIREPOWER-AP-PROCESSOR-MIB.my
index 9eff50d..e5ff9a3 100644
--- a/cisco-fp/CISCO-FIREPOWER-AP-PROCESSOR-MIB.my
+++ b/cisco-fp/CISCO-FIREPOWER-AP-PROCESSOR-MIB.my
@@ -58,8 +58,7 @@ IMPORTS
     CfprApProcessorQualArch,
     CfprApProcessorRuntimeHistThresholded,
     CfprApProcessorRuntimeThresholded,
-    CfprApProcessorUnitArch,
-    CfprApProcessorUnitId
+    CfprApProcessorUnitArch
         FROM CISCO-FIREPOWER-AP-TC-MIB;
 
 cfprApProcessorObjects MODULE-IDENTITY
diff --git a/cisco-fp/CISCO-FIREPOWER-AP-STORAGE-MIB.my b/cisco-fp/CISCO-FIREPOWER-AP-STORAGE-MIB.my
index 3efab8a..ac1dc4f 100644
--- a/cisco-fp/CISCO-FIREPOWER-AP-STORAGE-MIB.my
+++ b/cisco-fp/CISCO-FIREPOWER-AP-STORAGE-MIB.my
@@ -104,7 +104,6 @@ IMPORTS
     CfprApStorageLinkSpeed,
     CfprApStorageLocalDiskConfigFlexFlashRAIDReportingState,
     CfprApStorageLocalDiskConfigFlexFlashState,
-    CfprApStorageLocalDiskId,
     CfprApStorageLocalDiskMode,
     CfprApStorageLocalDiskPartitionType,
     CfprApStorageLunType,
@@ -117,13 +116,11 @@ IMPORTS
     CfprApStorageProtocol,
     CfprApStorageRaidBatteryOperabilityQualifier,
     CfprApStorageReadType,
-    CfprApStorageSyncAction,
     CfprApStorageSystemFsmCurrentFsm,
     CfprApStorageSystemFsmStageName,
     CfprApStorageSystemFsmTaskItem,
     CfprApStorageTechnology,
     CfprApStorageVDriveState,
-    CfprApStorageVirtualDriveId,
     CfprApStorageVsanRefSwitchId,
     CfprApVnicConfigIssues
         FROM CISCO-FIREPOWER-AP-TC-MIB;
diff --git a/cisco-fp/CISCO-FIREPOWER-AP-SW-MIB.my b/cisco-fp/CISCO-FIREPOWER-AP-SW-MIB.my
index 7f592f5..3078375 100644
--- a/cisco-fp/CISCO-FIREPOWER-AP-SW-MIB.my
+++ b/cisco-fp/CISCO-FIREPOWER-AP-SW-MIB.my
@@ -192,7 +192,6 @@ IMPORTS
     CfprApSwSspEthLanMonTransport,
     CfprApSwSspEthMonFsmCurrentFsm,
     CfprApSwSspEthMonFsmStageName,
-    CfprApSwSspEthMonFsmTaskFlags,
     CfprApSwSspEthMonFsmTaskItem,
     CfprApSwSspEthMonSrcPhyEpFsmCurrentFsm,
     CfprApSwSspEthMonSrcPhyEpFsmStageName,
