Kun i new/scte/: HMS026R16-1575478188239.txt
diff -ru tmp/scte/HMS154-1R6.mib new/scte/HMS154-1R6.mib
--- tmp/scte/HMS154-1R6.mib	2023-07-26 16:47:03.519364437 +0200
+++ new/scte/HMS154-1R6.mib	2023-07-26 16:50:36.147018794 +0200
@@ -15,7 +15,7 @@
 	  FROM SCTE-HMS-HEADENDIDENT-TC-MIB; 
 
 heDigitalCommonMIB MODULE-IDENTITY
-    LAST-UPDATED    "200807210000Z "
+    LAST-UPDATED    "200807210000Z"
     ORGANIZATION    "SCTE HMS Working Group"
     CONTACT-INFO    "SCTE HMS Subcommittee, Chairman 
                      mail to:  standards@scte.org "
diff -ru tmp/scte/HMS154-2R1.mib new/scte/HMS154-2R1.mib
--- tmp/scte/HMS154-2R1.mib	2023-07-26 16:47:06.018336856 +0200
+++ new/scte/HMS154-2R1.mib	2023-07-26 16:50:36.147018794 +0200
@@ -12,8 +12,8 @@
 		FROM IF-MIB
 	InetAddressType, InetAddress
 		FROM INET-ADDRESS-MIB
-	QAMModulationFormat
-	    FROM SCTE-HMS-HEADENDIDENT-TC;
+	QAMChannelModulationFormat
+	    FROM SCTE-HMS-HEADENDIDENT-TC-MIB;
 
 heDigitalQamMIB MODULE-IDENTITY
 	LAST-UPDATED "200710021200Z"
@@ -97,7 +97,7 @@
 	qamChannelFrequency
 		Unsigned32,
 	qamChannelModulationFormat
-		QAMModulationFormat,
+		QAMChannelModulationFormat,
 	qamChannelInterleaverLevel
 		INTEGER,
 	qamChannelInterleaverMode
@@ -122,7 +122,7 @@
 	::= { qamChannelEntry 1 }
 
 qamChannelModulationFormat  OBJECT-TYPE
-	SYNTAX     QAMModulationFormat
+	SYNTAX     QAMChannelModulationFormat
 	MAX-ACCESS read-only
 	STATUS     current
 	DESCRIPTION 
diff -ru tmp/scte/HMS154-3R15.mib new/scte/HMS154-3R15.mib
--- tmp/scte/HMS154-3R15.mib	2023-07-26 16:47:08.717307067 +0200
+++ new/scte/HMS154-3R15.mib	2023-07-26 16:50:36.147018794 +0200
@@ -17,7 +17,7 @@
 		FROM ENTITY-MIB   
 	DeviceEnableDisableValues, MpegErrorStatus, 
 	HePIDValue,VideoInputFrameRateType    
-		FROM SCTE-HMS-HEADENDIDENT-TC;
+		FROM SCTE-HMS-HEADENDIDENT-TC-MIB;
 
 mpegEncoderMIB MODULE-IDENTITY
 	LAST-UPDATED "200710031700Z"
